src/server/scripts/Commands/cs_ahbot.cpp

```diff
@@ -21,13 +21,18 @@
 #include "Language.h"
 #include "RBAC.h"
 
-uint32 const ahbotQualityIds[MAX_AUCTION_QUALITY] =
-{
-    LANG_AHBOT_QUALITY_GRAY, LANG_AHBOT_QUALITY_WHITE,
-    LANG_AHBOT_QUALITY_GREEN, LANG_AHBOT_QUALITY_BLUE,
-    LANG_AHBOT_QUALITY_PURPLE, LANG_AHBOT_QUALITY_ORANGE,
-    LANG_AHBOT_QUALITY_YELLOW
-};
+static std::unordered_map<AuctionQuality, uint32> const ahbotQualityLangIds =
```

Comment:
Could this be `std::array`?

Reply:
Uhm, I don't think so since I wanted to get rid of the order dependence and look up by `AuctionQuality` directly.
Maybe I'm missing something obvious here...

Reply:
fair, i guess `AuctionQuality` is not sorted